pools/kovan/metadata/0x6f3d561b203c69cdee90f2870ea58e5a7da460b2.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I think we probably need to add the start blocks of `ASSESSOR`, `SHELF`, and `FEED`, since those are used in the data sources of the subgraph.

Reply:
Never mind, we'll just start indexing them on `PoolCreated`

Reply:
I think we should add the start block of the root and then go from there (it happens first anyway)

Reply:
Looks like The Graph doesn't actually support setting the start block, it just uses the block when the data source gets created. So it will automatically use the block of the `PoolCreated` event. See: https://thegraph.com/docs/define-a-subgraph#data-source-templates

```Note: A new data source will only process the calls and events for the block in which it was created and all following blocks, but will not process historical data, i.e., data that is contained in prior blocks.```

We can even remove the `startBlock` key then. I'm not even sure why we currently have it in the pool configs, as we don't use it in the UI at all from what I can see.